### PR TITLE
Events within distance

### PR DESCRIPTION
**Pull request by noel-castillo**

### CODE REVIEW

File: findmyparadise_frontend/src/components/Events.vue [hunk 1]
```diff
@@ -62,7 +65,8 @@ export default {
   },
   mounted() {
     axios
-      .get("http://54.91.69.145:80/api/events")
```
Comment (tamera-brown) [inline findmyparadise_frontend/src/components/Events.vue:65]: Did you mean to change the api?

Comment (noel-castillo) [reply findmyparadise_frontend/src/components/Events.vue:65]: Ideally I need to have an api that takes in the listingId from props, but haven't figured it out yet. If the path I want is events/{listingId}/25, then would I do .get("http://54.91.69.145:80/api/events/15/25") or .get("http://localhost:8080/api/events/15/25")?

Comment (tamera-brown) [reply findmyparadise_frontend/src/components/Events.vue:65]: Use the api with the ip address

Comment (noel-castillo) [reply findmyparadise_frontend/src/components/Events.vue]: I sent commit with the proper route.

**Review by tamera-brown — APPROVED** [review]

**Pull request merged**